tests/setup-runtime-dependencies.sh

```diff
@@ -115,13 +115,18 @@ if [ ! -e $libInstallDir ]; then
 fi
 
 # Query runtime Id
-rid=`$dotnetCmd --info | grep 'RID:' | sed 's/^ *RID: *//g'`  
+rid=`$dotnetCmd --info | grep 'RID:' | sed 's/^ *RID: *//g'`
+#rid='ubuntu.14.04-x64'
```

Comment:
Is this a remainder of debugging code or a comment describing an example of RID?